scripts/global/pr_footer.md

```diff
@@ -1 +1 @@
-_This example was generated by [${SCRIPT}](${REPO_URL}/blob/main/scripts/${SCRIPT}) on ${DATE}. The original README follows._
+_This example was generated by [${SCRIPT}](${REPO_URL}/blob/scripts/${SCRIPT}) on ${DATE}. The original README follows._
```

Comment:
Well, keep _this_ change so we get the updated locations whenever we update any examples!